gravitee-apim-rest-api/gravitee-apim-rest-api-management-v2/gravitee-apim-rest-api-management-v2-rest/src/main/java/io/gravitee/rest/api/management/v2/rest/resource/api/ApiSubscriptionsResource.java

```diff
@@ -212,6 +215,28 @@ public Response exportApiSubscriptions(
             .build();
     }
 
+    @POST
+    @Path("/{subscriptionId}/_resumeFailure")
+    @Produces(MediaType.APPLICATION_JSON)
+    @Operation(
+        summary = "Change the status of a subscription",
```

Comment:
the goal is to pass a subscription from fail to started, not simply "change the status"